src/OpenRiaServices.Tools/Test/DomainServiceCatalogTests.cs

```diff
@@ -164,7 +179,12 @@ public void DomainServiceCatalog_Message_BadImageFormat()
             File.WriteAllText(assemblyFileName, "neener neener neener");
 
             ConsoleLogger logger = new ConsoleLogger();
+#if NETFRAMEWORK
             DomainServiceCatalog dsc = new DomainServiceCatalog(new string[] { assemblyFileName }, logger);
+#else
+            var dispatcher = new ClientCodeGenerationDispatcher();
+            DomainServiceCatalog dsc = new DomainServiceCatalog(new string[] { assemblyFileName }, logger, dispatcher);
```

Comment:
I think we should be able to remove the "dispatcher" parameter from DomainServiceCatalog which will allow several test to be changed back to the old implementation